.github/workflows/release.yaml

```diff
@@ -0,0 +1,58 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  create:
+    runs-on: "ubuntu-latest"
+    outputs:
+      upload_url: "${{ steps.release.outputs.upload_url }}"
+    steps:
+      - uses: "actions/create-release@v1"
+        id: "release"
+        env:
+          GITHUB_TOKEN: "${{ github.token }}"
+        with:
+          draft: "true"
+          tag_name: "${{ github.ref }}"
+          release_name: "${{ github.ref }}"
+
+  build-mac:
```

Comment:
do we need to make this job depend on goreleaser?

Reply:
It depends on `create`, which is all it needs to ensure the release is there for it to upload.

The `release` step in goreleaser also will create a release, but it [only creates](https://github.com/goreleaser/goreleaser/blob/61bead8989ea95dc7168efb58874ad07527071cb/internal/client/github.go#L128) if it doesn't exist, so I think these will play nicely with each other